core/rpc/src/system/error.rs

```diff
@@ -30,10 +37,17 @@ error_chain! {
 	}
 }
 
+const ERROR: i64 = 2000;
```

Comment:
Can you leave a brief comment what is this error exactly means and what one should do if they need to add a new error?

Reply:
Error code can be chosen arbitrarily (except for `-32768 to -32000` range, which is reseved). Currently we only return custom errors in `author` and `system` module. For author we start the error codes at `1k`, here at `2k`.
Perhaps we can move the range definition to some common place to avoid clashes. I can augment the PR if you think it's worth it.